extensions/goose/src/goose-recipe.ts

```
@@ -111,11 +111,20 @@ export class GooseRecipe implements Disposable {
     const decoded = Buffer.from(flowId, 'base64').toString('utf-8');
-    if (dirname(decoded) !== this.getBasePath()) throw new Error(`only support recipes in ${this.getBasePath()}`);
+    const normalised = this.normalizePath(decoded);
+    if (dirname(normalised) !== this.getBasePath()) throw new Error(`only support recipes in ${this.getBasePath()}`);
 
     return decoded;
   }
 
+  protected normalizePath(p: string): string {
+    // Remove extended-length prefix \\?\ on Windows
+    if (p.startsWith('\\\\?\\')) {
+      p = p.slice(4);
```

Biome (2.1.2)

[error] Assigning a function parameter is confusing.

The parameter is declared here:

Developers usually expect function parameters to be readonly. To align with this expectation, use a local variable instead.

(lint/style/noParameterAssign)